internal/impl/docker.go

```diff
@@ -108,70 +114,55 @@ downloaded and installed in the container. Do you want to proceed? [Y/n] `)
 		scanner.Scan()
 		text := scanner.Text()
 		if text != "" && text != "y" && text != "Y" {
-			return nil, fmt.Errorf("user bailed out")
+			return "", fmt.Errorf("user bailed out")
 		}
-		toInstall = append(toInstall, "github.com/ServiceWeaver/weaver-kube/cmd/weaver-kube@latest")
+		install = "github.com/ServiceWeaver/weaver-kube/cmd/weaver-kube@latest"
 	} else {
-		// Released tool binary that's not compiled to linux/amd64. Re-install
-		// it inside the container.
-		toInstall = append(toInstall, "github.com/ServiceWeaver/weaver-kube/cmd/weaver-kube@"+toolVersion)
-	}
-
-	if image == "" {
-		image = fmt.Sprintf("%s:%s", app.Name, depId[:8])
-	}
-
-	return &buildSpec{
-		image:     image,
-		files:     toCopy,
-		goInstall: toInstall,
-	}, nil
-}
-
-// buildImage builds a docker image with a given spec.
-func buildImage(ctx context.Context, spec *buildSpec) error {
-	fmt.Fprintf(os.Stderr, greenText(), fmt.Sprintf("Building image %s...", spec.image))
-	// Create:
-	//  workDir/
-	//    file1
-	//    file2
-	//    ...
-	//    fileN
-	//    Dockerfile   - docker build instructions
-	ctx, cancel := context.WithTimeout(ctx, time.Second*120)
-	defer cancel()
-
-	// Create workDir/.
-	workDir := filepath.Join(os.TempDir(), fmt.Sprintf("weaver%s", uuid.New().String()))
-	if err := os.Mkdir(workDir, 0o700); err != nil {
-		return err
+		// Install the currently running version of "weaver-kube" in the
+		// container.
+		install = "github.com/ServiceWeaver/weaver-kube/cmd/weaver-kube@" + toolVersion
 	}
-	defer os.RemoveAll(workDir)
 
-	// Copy the files from spec.files to workDir/.
-	for _, file := range spec.files {
-		workDirFile := filepath.Join(workDir, filepath.Base(filepath.Clean(file)))
-		if err := cp(file, workDirFile); err != nil {
-			return err
-		}
+	// Create a Dockerfile.
```

Comment:
nit: the comment seems somehow misplaced (confusing).

Reply:
Reworded a bit to make clearer.